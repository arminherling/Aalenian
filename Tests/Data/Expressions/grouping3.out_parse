AssignmentStatement: {
    Left: {
        DiscardLiteral: _
    }
    Right: {
        GroupingExpression: {
            BinaryExpression: {
                Left: {
                    GroupingExpression: {
                        BinaryExpression: {
                            Left: {
                                NumberLiteral: 1
                            }
                            Operation: Addition
                            Right: {
                                NumberLiteral: 2
                            }
                        }
                    }
                }
                Operation: Multiplication
                Right: {
                    NumberLiteral: 5
                }
            }
        }
    }
}
