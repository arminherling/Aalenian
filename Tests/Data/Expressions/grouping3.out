AssignmentStatement: {
    Left: {
        Discard: _
    }
    Right: {
        Grouping: {
            BinaryExpression: {
                Left: {
                    Grouping: {
                        BinaryExpression: {
                            Left: {
                                Number: 1
                            }
                            Operation: Addition
                            Right: {
                                Number: 2
                            }
                        }
                    }
                }
                Operation: Multiplication
                Right: {
                    Number: 5
                }
            }
        }
    }
}
